week-1-ts-ecosystem/w01e01-typescript-basics/src/main.ts

```diff
@@ -6,25 +8,26 @@
  *
  * */
 
-const user: any = {
-  name: 'Andy',
+const user: User = {
+  name: "Andy",
   age: 30,
-  email: 'andy@mail-me-tommorow.com',
+  email: "andy@mail-me-tommorow.com",
   address: {
-    street: 'Strange Alley',
+    street: "Strange Alley",
     no: 23,
   },
-}
+};
 
-function hasAddress(user: any): any {
-  return Boolean(user.address)
+function hasAddress(user: User): boolean {
+  return Boolean(user.address);
 }
 
-function hasGivenAge(requiredAge: any): any {
-  return (user: any): any => user.age >= requiredAge
+function hasGivenAge(requiredAge: number): Function { 
+  return (user: User):boolean => user.age >= requiredAge;
+
 }
 
-const isAdult = hasGivenAge(18)
+const isAdult = hasGivenAge(18);
```

Comment:
Te funkcje też mogłby zamieszkać w oddzielnym module (pliku) np. jako `user.validators.ts` ?

Reply:
to nie zostało przeniesione @kubaGalinski .

Zgubiłeś czy decyzja świadoma? :)